backend/Startup.cs

```diff
@@ -24,6 +27,9 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllersWithViews();
+            services.AddDbContext<MovieContext>(builder => builder.UseSqlServer("DevConnection"));
```

Comment:
why are there so many contexts?? what's the idea?

Reply:
I tried to generate a Controller for Calendar, but I gave up on the idea and I forgot to remove the DbContext for Calendar, sorry.